ssl/statem/extensions.c

```diff
@@ -54,13 +52,14 @@ typedef struct extensions_definition_st {
      */
     int (*init)(SSL *s, unsigned int context);
     /* Parse extension sent from client to server */
-    int (*parse_ctos)(SSL *s, PACKET *pkt, int *al);
+    int (*parse_ctos)(SSL *s, PACKET *pkt, X509 *x, size_t chain, int *al);
```

Comment:
"chain" is the wrong name.  It's really numcerts or xcount.  And it should be an int not a size_t.